drivers/staging/vc04_services/bcm2835-audio/bcm2835.c

```diff
@@ -384,8 +385,12 @@ static int snd_bcm2835_alsa_probe(struct platform_device *pdev)
 		if (!of_property_read_bool(dev->of_node, "brcm,disable-hdmi"))
 			set_hdmi_enables(dev);
 
-		// In this mode, always enable analog output
-		enable_headphones = true;
+		// In this mode, enable analog output by default
+		disable_headphones = 0;
```

Comment:
nit pick: Just use ```u32 disable_headphones = 0;``` within this scope (defined at the beginning of it), instead of having the variable definition and initialisation separate.

Reply:
This was a conscious choice because I like the fact that the initialisation is closer to where the value is used - it requires less scrolling back to discover the default value - and checkpatch is happy with it.

Reply:
But if you are aware of similar cases where upstream devs have objected then I am willing to change it,

Reply:
Is 3 lines really that much to scroll back up?
```
	if (!enable_compat_alsa) {
		// In this mode, enable analog output by default
		u32 disable_headphones = 0;

		if (!of_property_read_bool(dev->of_node, "brcm,disable-hdmi"))
			set_hdmi_enables(dev);

		of_property_read_u32(dev->of_node,
				     "brcm,disable-headphones",
				     &disable_headphones);
		enable_headphones = !disable_headphones;
```
It's not used outside of this clause, so why define it at the start of the function?

Reply:
Because that seems to be the kernel style - old-school C. But checkpatch is happy with either so the updated version goes with the block-scoped `disable_headphones`.